lib/plan-finder.js

```diff
@@ -259,10 +266,56 @@ class PlanFinder {
         if (!this.isMimeType(outputInstructions.type)) {
             throw new RenditionFormatUnsupportedError(`outputInstructions.type (${outputInstructions.type}) is not propertly formatted`);
         }
-        const plan = this.BFS(source, outputInstructions);
+
+        let plan;
+        // get image metadata and check for "Orientation" property
+        const metadata = await this.getImageMetadata(source.path);
```

Comment:
this needs to be done after we run BFS, inside `formPlan`.

We only need to adjust the orientation if the file is going straight to sensei. If the file is getting converted or resized, it will get orientation applied in that step already